docs/lib/auth/fragments/native_common/signout/common.md

```diff
@@ -7,3 +7,9 @@ Calling signOut without any options will just delete the local cache and keychai
 
 <inline-fragment platform="android" src="~/lib/auth/fragments/android/signout/20_global_signout.md"></inline-fragment>
 <inline-fragment platform="ios" src="~/lib/auth/fragments/ios/signout/20_global_signout.md"></inline-fragment>
+
+Calling signout with `globalSignOut = true` will invalidate all the Cognito User Pool token of the signed in user. If the user is signed in into a device, they wont be authorized to perform task that require a valid token when a global signout is called from some other device.
```

Comment:
signed into a device

Reply:
perform a task that requires a valid token

Reply:
Fixed